decidim-budgets/app/helpers/decidim/budgets/projects_helper.rb

```diff
@@ -23,6 +23,10 @@ def current_order_budget_percent
       def current_order_can_be_checked_out?
         current_order&.can_checkout?
       end
+
+      def projects_base_url
+        URI.parse(root_url).tap { |uri| uri.query = nil } .to_s
```

Comment:
should we also ignore he `uri.fragment`?

Reply:
AFAIK that part of the URI shouldn't reach the server. If you navigate to `https://example.org/path/script?querystring#fragment` you could see in your browser's Network tab that the HTTP request only asks for `/path/script?querystring`.

Anyway, it's so simple that I will change it to `uri.query = uri.fragment = nil` to avoid unexpected issues. :)

Reply:
fixed and rebased to the develop branch!